templates/header.redhat.html

```diff
@@ -1,35 +1,10 @@
-{% extends "base.html" %}
+<!DOCTYPE html>
 
-{% block distribution_instruction %}
+<html lang="en">
 
-  <p>
-  To use this repository, run the following command:
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="refresh" content="1; url='https://www.jenkins.io/doc/book/installing/linux/'" />
+</head>
 
-  <pre class="text-white bg-dark">
-
-  sudo wget -O /etc/yum.repos.d/{{artifactName}}.repo {{ web_url }}/{{artifactName}}.repo
-  sudo rpm --import {{ web_url }}/{{organization}}-2023.key
-  </pre>
-
-  <p>
-  If you've previously imported the key from Jenkins, the <code>rpm --import</code> will fail because
-  you already have a key. Please ignore that and move on.
-
-  <pre class="text-white bg-dark">
-
-  yum install fontconfig java-17-openjdk
```

Comment:
Where is the reference to `fontconfig` on https://www.jenkins.io/doc/book/installing/linux?

Reply:
Thanks for catching that.  Added to the documentation in https://github.com/jenkins-infra/jenkins.io/pull/6721